e2e/mock-api/project.json

```diff
@@ -4,7 +4,7 @@
   "projectType": "application",
   "targets": {
     "build": {
-      "executor": "@nrwl/node:build",
+      "executor": "@nrwl/node:webpack",
```

Comment:
I did an `nx` update and there was a bug in `nx` that released a major instead of a minor and this was changed as a result.